tests/compile-test.rs

```diff
@@ -331,10 +335,10 @@ fn compile_test() {
     prepare_env();
     let mut config = default_config();
     run_ui(&mut config);
-    run_ui_test(&mut config);
-    run_ui_toml(&mut config);
-    run_ui_cargo(&mut config);
-    run_internal_tests(&mut config);
+    // run_ui_test(&mut config);
+    // run_ui_toml(&mut config);
+    // run_ui_cargo(&mut config);
+    // run_internal_tests(&mut config);
```

Comment:
If you want to only run a specific UI test you can do so with

```
TESTNAME=eta cargo uitest
```

This will only run UI tests containing the string "eta".

Reply:
thanks @flip1995 please do not review yet :sweat_smile: I commented those out to avoid all the `Running 0 tests`  that I still get using `TESTNAME`. This won't be there at the end, no worries